recipes/datreant.core/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.7.0" %}
+
+package:
+  name: datreant.core
+  version: "0.7.0"
+
+source:
+  git_url: https://github.com/datreant/datreant.core.git
+  git_tag: release-{{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+    - asciitree
+    - pathlib2
+    - scandir
+    - six
+    - fuzzywuzzy
+    - pytest
+    - datreant
+
+  run: &package_deps
```

Comment:
I can also use jinja templates if that is ok with you. I would just like to avoid having to retype package names 3 times. Using yaml anchors reduces the chance of me making an error with the dependency packages.